### PR TITLE
Remove user role from discussion profile page

### PR DESCRIPTION
**Pull request by saharm1**

### CODE REVIEW

**Comment by saharm1** [comment]
@tkim135

**Comment by caesar2164** [comment]
Great, now see if you can figure out how to hide it through SASS/styling! :)

**Comment by saharm1** [comment]
@caesar2164 done!

File: lms/static/sass/discussion/_layouts.scss [hunk 1]
```diff
@@ -22,7 +22,8 @@
     }
 
     .user-roles {
-        font-size: $forum-small-font-size;
+        display: none;
+	font-size: $forum-small-font-size;
```
Comment (caesar2164) [inline lms/static/sass/discussion/_layouts.scss]: hmm, looks like you removed a space here

**Comment by stvstnfrd** [comment]
@caesar2164 Shouldn't we be making this change in the theme?

**Comment by caesar2164** [comment]
yep, I already told @saharm1, to do that now.

**Comment by caesar2164** [comment]
@saharm1, now that the theme PR is open, let's close this one.

**Pull request closed without merging**